README.md

```diff
@@ -258,10 +258,25 @@ expect([ 1, 2, 3 ]).toExclude(4)
 Asserts the given `object` contains all keys and values in `value`, recursively. The `comparator` function, if given, should compare two objects and either `return false` or `throw` if they are not equal. It defaults to `assert.deepEqual`.
 
 ```js
+expect({ a: 1, b: 2 }).toInclude('a')
```

Comment:
i'm not really a fan of overloading APIs to take multiple types. I'd rather see a separate `.toIncludeKeys` that takes an array of strings, for example.